scripts/alert_emails/campaign-monitor.ts

```diff
@@ -81,6 +94,26 @@ class CampaignMonitor {
 
     return response.data;
   }
+
+  async fetchTransactionalStats(
+    group: string,
+    from: Date,
+    to: Date,
```

Comment:
Per my comment elsewhere I wonder if we don't need to filter by date anymore, in which case you can remove these or make them optional or something?

Reply:
That makes sense, the default for `from` is 29 days ago, which should be good enough.